proto/axelar/nexus/v1beta1/types.proto

```diff
@@ -17,23 +17,23 @@ message MaintainerState {
       [ (gogoproto.nullable) = false ];
   axelar.utils.v1beta1.Bitmap incorrect_votes = 3
       [ (gogoproto.nullable) = false ];
+  string chain = 4
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
 }
 
 // ChainState represents the state of a registered blockchain
 message ChainState {
   reserved 4; // total was removed in v0.13
+  reserved 2; // maintainers was removed in v0.24
```

Comment:
The underlying values weren't deleted before when it was deprecated, so we delete those during migration?

Reply:
yes